Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptTypesService.java

```diff
@@ -0,0 +1,47 @@
+package edu.asu.conceptpower.app.core.impl;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.app.core.IConceptTypesService;
+
+/**
+ * This helper class is used for identifying the concept types based on the
+ * concept id.
+ * 
+ * @author karthikeyanmohan
+ *
+ */
+@Service
+public class ConceptTypesService implements IConceptTypesService {
+    
+    /**
+     * This enum contains the list of valid concept types.
+     * 
+     * @author karthikeyanmohan
+     *
+     */
+    public enum ConceptTypes {
+        GENERIC_WORDNET_CONCEPT, SPECIFIC_WORDNET_CONCEPT, LOCAL_CONCEPT
+    }
+    
+    /**
+     * This method returns the concept types based on the id. If id is null then
+     * null value is returned by the method.
```

Comment:
"concept types" should a) be singular, and b) it's not really a concept type (we have those and they are something else), but the type of id used to refer to a concept.